editor.planx.uk/src/theme.ts

```diff
@@ -13,8 +13,11 @@ import createPalette, {
   PaletteOptions,
 } from "@mui/material/styles/createPalette";
 import { deepmerge } from "@mui/utils";
+import { TeamTheme } from "@opensystemslab/planx-core/types";
+import { getContrastTextColor } from "styleUtils";
 
 const DEFAULT_PRIMARY_COLOR = "#0010A4";
+const DEFAULT_TONAL_OFFSET = 0.2;
```

Comment:
No change here, this is just the MUI default we're currently using.

Docs: https://mui.com/material-ui/customization/palette/#tonal-offset